src/runtime/composables/useApiData.ts

```diff
@@ -78,10 +78,15 @@ export function _useApiData<T = any>(
 
   const _fetchOptions = reactive(fetchOptions)
 
+  const _headers = {
+    ...headers,
```

Comment:
`headers` may be a reactive object, that's why we call `toValue` below. To unwrap the reactive object.